docs/podman-network-inspect.1.md

```diff
@@ -0,0 +1,50 @@
+% podman-network-inspect(1)
+
+## NAME
+podman\-network-inspect- Inspect one or more CNI networks
+
+## SYNOPSIS
+**podman network inspect**  [*network* ...]
+
+## DESCRIPTION
+Display the raw (JSON format) CNI network configuration. This command is not available for rootless users.
```

Comment:
On the fence.  If someone's fresh to container land, they might not know CNI.  Maybe "Container Network Interface (CNI) configuration" instead of "CNI network configuration".

Regardless, isn't "CNI network configuration" redundant?  "CNI configuration".

If you use either of these, please change throughout.